### PR TITLE
Adding Design and Architecture Details

### PR DESCRIPTION
**Pull request by VikasKumarC**
Signed-off-by: Vikas Kumar Choudhary <vikas.kumar@seagate.com>

# Problem Statement
-  Problem statement

# Design
-  For Bug, Describe the fix here.
-  For new test addition post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is formatted (e.g. using pycharm in-built formatter)

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed
-  [ ] Attach test execution logs

# Review Checklist 
  Checklist for Author
-  [ ] JIRA number/GitHub Issue added to PR
-  [ ] PR is self reviewed
-  [ ] Jira and state/status is updated and JIRA is updated with PR link
-  [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe

-----
[View rendered docs/Architecture_and_Design.md](https://github.com/VikasKumarC/corio/blob/Design_and_flow/docs/Architecture_and_Design.md)

### CODE REVIEW

**Review by akshaym99 — APPROVED** [review]

**Review by azaharkhan — APPROVED** [review]

File: docs/Architecture_and_Design.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+Why CORIO (KOR-I-O) 
+
+Need to check IO Stability: Sustainability of S3 Ops with IO for expected longer duration.
+
+Need for a tool which gives consistent outcome of where do we stand in terms of IO Path stability.
+Hereby needed single aggregator / umbrella tool.
+
+    * This tool has capability to embed multiple IO tools (S3bench, WARP, etc.) and benefits from their specific capabilities.
```
Comment (ketanarlulkar) [inline docs/Architecture_and_Design.md]: Untab this, so it will be normal list

Comment (ketanarlulkar) [reply docs/Architecture_and_Design.md]: Same for all lists

File: docs/Architecture_and_Design.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+Why CORIO (KOR-I-O)
```
Comment (ketanarlulkar) [inline docs/Architecture_and_Design.md]: Use headers [github-link](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#headings)

**Review by kulrk — APPROVED** [review]

File: docs/Architecture_and_Design.md (hunk 2 above)
Comment (VikasKumarC) [reply docs/Architecture_and_Design.md]: Done

File: docs/Architecture_and_Design.md (hunk 1 above)
Comment (VikasKumarC) [reply docs/Architecture_and_Design.md]: Done, Please check

**Review by DhobaleVishal — APPROVED** [review]

**Review by ketanarlulkar — APPROVED** [review]

**Pull request merged**